data/category-acg

```diff
@@ -1,5 +1,30 @@
+include:abema
+include:acfun @cn
```

Comment:
1. a category-acg-cn would be better
2. corp. like acfun may also have some sites mainly serving the oversea users. Labelling a sub-category does not practically make sense, see the point 1

Reply:
Then how about have it split to category-acg-cn and category-acg-!cn?